### DISCUSSION TITLE
希望组件的在线仓库能添加github更新源

### DISCUSSION THREAD

**Discussion by xxldm** [the1812/Bilibili-Evolved#4159] [功能建议 / Ideas]
希望组件的在线仓库能添加github更新源。不然本体更新之后更新日志里面的内容不生效，我都不知道是没有更新好，还是cdn有缓存

**Comment by the1812** [comment]
jsDelivr 源有 24h 的缓存, 一般公告发出后一天才能收到更新.

通用设置里就可以设置更新源, 不过不保证高版本的组件能在旧版本体上运行.

**Reply by xxldm** [reply]
刚新下了一个组件确实是github的，那我应该是以前先下的组件，后改的更新源，导致已下载的组件URL没有更新？
我已经在油猴->存储里手动批量改成了github了，组件已经更新好了，没有问题了。
谢谢，辛苦了！
